genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/DirectoryManifest.java

```diff
@@ -76,24 +76,14 @@
     private final int numDirectories;
     private final long totalSizeOfFiles;
 
-    /**
-     * Create a manifest from the given job directory.
-     *
-     * @param directory The job directory to create a manifest from
-     * @throws IOException If there is an error reading the directory
-     */
-    public JobDirectoryManifest(final Path directory) throws IOException {
```

Comment:
oh oh oh, this is two different changes in one commit! You used the word "also"... 
Just giving you a hard time you don't actually have to change this :-)